### PR TITLE
Added What’s New in Swift 4 by Example tutorial.

### PR DESCRIPTION
**Pull request by KaramazovShogun**
Added What’s New in Swift 4 by Example tutorial.

### CODE REVIEW

File: Issues/Week188.md [hunk 1]
```diff
@@ -24,4 +25,5 @@
 
 **Credits**
 
-*  [rbarbosa](https://github.com/rbarbosa), [LisaDziuba](https://github.com/lisadziuba)
+*  [rbarbosa](https://github.com/rbarbosa), [LisaDziuba](https://github.com/lisadziuba), [KaramazovShogun](https://twitter.com/KaramazovShogun)
```
Comment (dkhamsing) [inline Issues/Week188.md]: This should be your GitHub profile

**Comment by KaramazovShogun** [comment]
Added Github account.

File: Issues/Week188.md (hunk 1 above)
Comment (KaramazovShogun) [inline Issues/Week188.md]: Added GitHub account.

**Comment by dkhamsing** [comment]
Thx!

**Comment by KaramazovShogun** [comment]
Thank you for adding my tutorial to your newsletter - much appreciated!

File: Issues/Week188.md (hunk 1 above)
Comment (KaramazovShogun) [inline Issues/Week188.md]: Thank you for adding my tutorial to your newsletter - much appreciated!

**Pull request merged**